README.md

```diff
@@ -206,9 +206,10 @@ what to put for each step.
    name of the domain, but at the time of installation the domain (or
    _record set_) does not _actually_ have to be configured in _Route 53_
 
-1. Next you are prompted for AWS credentials. These can be made
-   specifically for the instance, or for a dev instance you could use
-   your own.
+1. Next you are prompted for the default AWS region to be used and for
```

Comment:
I think you should get rid of the word `default` here and in the prompt in install_bootstrap

Reply:
Can you remove the word `default`?